controllers/storagecluster/storagecluster_controller.go

```diff
@@ -129,6 +130,21 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	}
 
+	ocsClientOperatorStatusPredicate := predicate.Funcs{
```

Comment:
The predicate name should reflect the entity/rule it is applying. 
```suggestion
	storageConsumerStatusPredicate := predicate.Funcs{
```